lib/index.js

```diff
@@ -1,35 +1,55 @@
 const playerNameInputs = document.querySelectorAll('.player-name-inputs');
 const domPlayerDivs = document.querySelectorAll('.player-names');
 const domCategoryDivs = document.querySelectorAll('.category-text');
+const ddWagerInput = document.querySelector('.wager-dd-text-input');
+const wagerErrorText = document.querySelector('.dd-wager-error-message');
+const answerInput = document.querySelector('.answer-text-input');
+const answerResponseText = document.querySelector('.answer-message-text');
 
-// store these in game class? having trouble accessing them here.
+// store these in classes. having trouble accessing them here.
 let currentClue;
 let currentRoundObj = {};
 
+let game = new Game();
+
 document.querySelector('.restart-button').addEventListener('click', resetGame);
 document.querySelector('.start-button').addEventListener('click', startGame);
 // document.querySelector('.player-name-inputs').forEach((playerName) => {
 //   playerName.on('input', checkInputs);
 // });
-// document.querySelector('.submit-dd-wager-button').addEventListener('click', submitDailyDoubleWager);
-// document.querySelector('.wager-dd-text-input').addEventListener('input', checkInputs);
+document.querySelector('.submit-dd-wager-button').addEventListener('click', submitDailyDoubleWager);
 // document.querySelector('.submit-wagers-button').addEventListener('click', submitFinalWagers);
 // document.querySelector('.wager-text-input').forEach((wager) => {
 //   wager.addEventListener('input', checkInputs);
 // });
-// document.querySelector('.submit-answer-button').addEventListener('click', submitAnswer);
-// document.querySelector('.answer-text-input').addEventListener('input', checkInputs);
+document.querySelector('.submit-answer-button').addEventListener('click', submitAnswer);
 document.querySelectorAll('.clue-box').forEach((box) => {
   box.addEventListener('click', displayClue);
 });
 
 function displayClue() {
```

Comment:
I had added code to invoke the validateAnswer method in the Clue class from here. Will need to review and figure out which approach to move forward with.